records_mover/creds/creds_via_env.py

```diff
@@ -15,6 +15,7 @@
 class CredsViaEnv(BaseCreds):
     def _gcp_creds(self, gcp_creds_name: str,
                    scopes: Iterable[str]) -> 'google.auth.credentials.Credentials':
+        # TODO: Make this the default one?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker